config/manager/manager.yaml

```diff
@@ -103,7 +103,7 @@ spec:
             cpu: 10m
             memory: 64Mi
         env:
-        - name: WATCH_NAMESPACE
+        - name: EVEREST_NAMESPACE
```

Comment:
This namespace will be primary, yet during the deployment we need to set WATCH_NAMESPACES=namespace1,namespace2 to run reconciliation logic there